webpack.parts.js

```diff
@@ -4,7 +4,7 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const TerserPlugin = require('terser-webpack-plugin')
 
-const publicPath = '/'
+const publicPath = ''
```

Comment:
Unfortunately, this will break the production build because webpack won't be able to find some public assets (fonts, etc), so could you also change this back to '/'?